core/templates/domain/collection/collection-update.service.ts

```diff
@@ -75,6 +75,7 @@ angular.module('oppia').factory('CollectionUpdateService', [
 
     // Applies a property change to a collection node. See _applyChanges() for
     // details on the other behavior of this function.
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we need this? Do we not need the below variable? If we don't need why not remove it?

Reply:
Removed function.